app/jobs/github_jobs/closed_job.rb

```diff
@@ -0,0 +1,9 @@
+module GithubJobs
+  class ClosedJob < ApplicationJob
+    queue_as :closed
```

Comment:
does this queue even exist? By default sidekiq uses just the `default` queue. BTW it's not recommended by sidekiq to have lot of queues, I think `default` `high` and `low` could be enough for now.  https://github.com/mperham/sidekiq/wiki/Advanced-Options#queues